app/uk/gov/hmrc/nisp/views/includes/nirecordtaxyear.scala.html

```diff
@@ -86,7 +88,13 @@
                     classThreePayableWithPenalty <- niRecordTaxYear.classThreePayableByPenalty
                 ){
                         <p class="contributions-header">@Messages("nisp.nirecord.gap.youcanmakeupshortfall")</p>
-                        <p>@Html(Messages("nisp.nirecord.gap.payvoluntarycontrib",NispMoney.pounds(niRecordTaxYear.classThreePayable.getOrElse(0)),Dates.formatDate(classThreePayableWithPenalty.localDate),Dates.formatDate(classThreePayableByDate.localDate)))</p>
+
+                        @if(niRecordTaxYear.checkCutOffDate(Some(classThreePayableByDate),new LocalDate())) {
```

Comment:
The value is dynamic, how would you put it in the model without passing it in upon construction?
If you hard code the value in there it's not unit testable.